backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportHeaderGenerator.kt

```diff
@@ -52,79 +52,88 @@ internal class ObjCExportTranslatorImpl(
         val mapper: ObjCExportMapper,
         val namer: ObjCExportNamer,
         val warningCollector: ObjCExportWarningCollector,
-        val objcGenerics: Boolean
+        val objcGenerics: Boolean,
+        val relaxedMode: Boolean
 ) : ObjCExportTranslator {
 
     private val kotlinAnyName = namer.kotlinAnyName
 
-    override fun generateBaseDeclarations(): List<ObjCTopLevel<*>> {
-        val stubs = mutableListOf<ObjCTopLevel<*>>()
-
-        stubs.add(objCInterface(namer.kotlinAnyName, superClass = "NSObject", members = buildMembers {
-            +ObjCMethod(null, true, ObjCInstanceType, listOf("init"), emptyList(), listOf("unavailable"))
-            +ObjCMethod(null, false, ObjCInstanceType, listOf("new"), emptyList(), listOf("unavailable"))
-            +ObjCMethod(null, false, ObjCVoidType, listOf("initialize"), emptyList(), listOf("objc_requires_super"))
-        }))
+    override fun generateBaseDeclarations(): List<ObjCTopLevel<*>> = buildTopLevel(relaxedMode) {
```

Comment:
Adding wrappers (for `buildTopLevel` and `buildMembers`) that don't accept `relaxedMode` as a parameter value but obtain it from property instead would simplify the code and help to avoid affecting places like this by changes to the configuration.

Reply:
✅